types/setting.go

```diff
@@ -255,6 +261,33 @@ var (
 		Default:     "300",
 	}
 
+	SettingDefinitionFailedBackupAutoDeleteion = SettingDefinition{
+		DisplayName: "Failed Backup Auto-Deletion",
+		Description: "This setting allows Longhorn to delete the backup resource that was failed automatically.\n\n" +
+			"WARNING: \n\n" +
+			"  - The failed backup will be removed in **BackupstorePollInterval** (default: 5 mintues) and you can only check the backup failed from event log.\n" +
+			"  - Keeping lots of the failed backups might affect the performance for backup volumes, upgrading or uninstallating longhorn system." +
+			"  - Setting 0 to disable **Backupstore Poll Interval** also means to disable failed Backup auto-deletion.",
+		Category: SettingCategoryBackup,
+		Type:     SettingTypeBool,
+		Required: true,
+		ReadOnly: false,
+		Default:  "true",
+	}
+
+	SettingDefinitionFailedBackupTTL = SettingDefinition{
+		DisplayName: "Failed Backup Time to Live",
+		Description: "In seconds. This setting determines how long Longhorn will keep the backup resource that was failed.\n\n" +
+			"WARNING: \n\n" +
```

Comment:
The `WARNING` looks not necessary. The following two lines are just the usage.

Reply:
Agree, Use `Note` instead of `Warning`, WDYT? Thanks.